setup.py

```diff
@@ -20,7 +20,11 @@
 setup(
     name='optimizely-sdk',
     version=__version__,
-    description="SDK for Optimizely's Full Stack Python project.",
+    description='Python SDK for Optimizely X Full Stack.',
+    long_description='Optimizely X Full Stack is A/B testing and feature management for product development teams. '
```

Comment:
A more standard approach to this is to set the value of `long_description` to the contents of a `README.md` (or similar) prepended to the contents of the `CHANGELOG.md` (or similar).

e.g.

```
with open('README.md') as _file:
  README = _file.read()
with open('CHANGELOG.md') as _file:
  CHANGELOG = _file.read()

setup(
  ...
  long_description=README + '\n\n# Change Log: \n' + CHANGELOG
  ...
)
```

Any thoughts?

Reply:
Yeah i noticed that in some implementations.

How about we do the message mentioned here + README + CHANGELOG?

Reply:
SGTM

Reply:
Don't forget to add those files in the `MANIFEST.in` so they're included in the package distribution. Installs will fail if without them.

Reply:
Good call. Need README there.